src/components/labelPrinting/LabelPrintingForm.vue

```diff
@@ -0,0 +1,194 @@
+              required
+            ></b-form-input>
+          </b-form-group>
+
+          <b-form-group
+            id="printer_choice_group"
+            label="Choice of Printer:"
+            label-for="printer_choice"
+            description="The printer to print the labels."
+          >
+            <b-form-select
+              id="printer_choice"
+              v-model="form.selectedPrinterName"
+              :options="printerOptions"
+              value-field="text"
+              required
+            ></b-form-select>
+          </b-form-group>
+
+          <b-form-group
+            id="copies_group"
+            label="Number of copies per label:"
+            label-for="copies"
+            description="Number of copies of each label you would like to print."
+          >
+            <b-form-input
+              id="copies"
+              v-model="form.copies"
+              type="number"
+              :min="1"
+              :max="10"
+              placeholder="Please select a number"
+              required
+            ></b-form-input>
+          </b-form-group>
+
+          <b-button id="submitButton" type="submit" variant="primary">Print Labels</b-button>
+          <b-button id="resetButton" type="reset" variant="danger" class="float-left"
+            >Reset</b-button
+          >
+        </b-form>
+      </b-col>
+      <b-col>
+        <div>
+          <b-card
+            title="List of barcodes to be printed:"
+            tag="article"
+            style="max-width: 20rem"
+            class="mb-2"
+          >
+            <b-card-text>
+              <ul id="list-barcodes-to-print">
+                <li v-for="(item, index) in suffixedBarcodes()" :key="index + 1">{{ item }}</li>
+              </ul>
+            </b-card-text>
+          </b-card>
+        </div>
+      </b-col>
+    </b-row>
+  </b-container>
+</template>
+
+<script>
+import SuffixList from '@/config/SuffixList'
+import { mapActions } from 'vuex'
+
+const defaultForm = () => ({
+  barcode: null,
+  selectedSuffix: null,
+  selectedNumberOfLabels: null,
+  selectedPrinterName: null,
+  copies: null,
+})
+
+export default {
+  name: 'LabelPrintingForm',
+  data() {
+    return {
+      form: defaultForm(),
+      suffixOptions: [],
+      printerOptions: [],
+      show: true,
+    }
+  },
+  created() {
+    this.setSuffixOptions()
+    this.setPrinterNames()
+  },
+  methods: {
+    setSuffixOptions() {
+      let suffixOptions = SuffixList.map((obj) => ({
+        text: obj.one_character_name,
+      }))
+      this.suffixOptions = suffixOptions
+    },
+    setPrinterNames() {
+      let printerOptions = this.$store.getters.printers.map((name) => ({
+        text: name,
+      }))
+      this.printerOptions = printerOptions
+    },
+    suffixedBarcodes() {
+      var listSuffixedBarcodes = []
+
+      if (this.form.barcode && this.form.selectedSuffix && this.form.selectedNumberOfLabels <= 9) {
```

Comment:
There is a min and max on the field but we are also validating it? The validation is that it is <=9 but the max is 10.

Reply:
Min max don't actually limit the inputs, and only get validated when you trigger the native HTML5 validation via triggering a form submit. All they do outside of this context is limit the range of the spinner.

Reply:
Ye the min and max on b-form-input doesn't prevent somebody from typing in directly "100". The max of 10 is for the number of copies, so a different field which doesn't have other validation